apps/explorer/src/ui/TransactionType.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    type ExecutionStatusType,
+    type TransactionKindName,
+} from '@mysten/sui.js';
+
+import { ReactComponent as ContentSuccessStatus } from '../assets/SVGIcons/12px/Check.svg';
+import { ReactComponent as ContentFailedStatus } from '../assets/SVGIcons/12px/X.svg';
```

Comment:
Use the local icons instead, which is supposed to be a more curated set. Ensure that you replace any colors with `currentColor` when creating the icons as well, so that they can easily be themed